### ISSUE TITLE
(OH GOD OH FUCK) HOLODECK STATUS IS CURRENTLY HOMONGOFUCKED

### ISSUE THREAD

**Issue by san7890** [tgstation/tgstation#56561]
## Reproduction: FUCK FUCK FUCK FUCK FCUK
I believe this is related to PR #55645, as it was the most recent thing to fuck with the Holodeck and how it swaps between simulation to simulation.
Basically, it was a highpop round in Bagil, and as the way usually goes, it got to up to around 70-80% Time Dilation. What happened, when I tried to make a vaccine for a particularly bad virus that round at the Holodeck, the simulations... would not clear.

The Start:
![image](https://user-images.githubusercontent.com/34697715/106407339-095c9100-63f9-11eb-9058-7d6a23cc0aa7.png)

The Middle:
![image](https://user-images.githubusercontent.com/34697715/106407377-1d07f780-63f9-11eb-9461-01fae9e59610.png)

The End (as I shot the 10x fuel cans using a laser gun from a previous simulation):
![image](https://user-images.githubusercontent.com/34697715/106407388-25603280-63f9-11eb-8b42-1f1bd62a1812.png)

The (Jannie) End (when you stack turfs with a shitload of plasma): 
![image](https://user-images.githubusercontent.com/34697715/106407541-8be55080-63f9-11eb-8c15-cf1d91271858.png)

Basically, the very simple functionality of the holodeck clearing the items before rendering a new holodeck simulation got absolutely fucked when it was hit with the high time dilation beams that Bagil tends to have. Please fix pretty fast, the time dilation gets worse and worse the more you try and clear out the holodecks.

**Comment by oklews** [comment]
fun fact, it breaks no matter what.

**Comment by oklews** [comment]
I tested it round start before it got laggy and it STILL broke

**Comment by Kylerace** [comment]
whoopsie
